### PR TITLE
Shows gobble data in the dashboard

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

Get the app in a state ready for gobble data

## Changes

Only shows gobble data when manually navigating to a bus url with a date past the current maximum.

Nobody can reach gobble data through the UI, but could by changing `startDate=2024-01-22` if they wanted to.

Adds a new warning banner to the bottom explaining that we parsed this data ourselves

### CODE REVIEW

**Comment by mathcolo** [comment]
whoa! is this on beta?

**Comment by devinmatte** [comment]
We have a version of this that includes commuter rail on beta. I can push this branch up instead if we want

**Comment by mathcolo** [comment]
That would be great, would you mind?

**Comment by devinmatte** [comment]
@mathcolo on beta now

File: server/chalicelib/s3.py [hunk 1]
```diff
@@ -28,16 +28,27 @@ def upload(key, bytes, compress=True):
...
+def is_bus(stop_id: str):
     return ("-0-" in stop_id) or ("-1-" in stop_id)
 
 
-def download_one_event_file(date, stop_id):
+def get_live_folder(stop_id: str):
+    if is_bus(stop_id):
+        return "daily-bus-data"
+    else:
+        return "daily-data"
```
Comment (hamima-halim) [inline server/chalicelib/s3.py:39]: the other folders in `Events-live/` are `daily-cr-data/` and `daily-rapid-data/`! `daily-data` is a subfolder of `Events/`

File: server/scripts/generate_line_files.py [hunk 2]
```diff
@@ -20,52 +20,70 @@
     "CR-Needham",
     "CR-Newburyport",
     "CR-Providence",
-    "CR-Foxboro",
```
Comment (hamima-halim) [inline server/scripts/generate_line_files.py:23]: why remove foxboro?

File: server/chalicelib/s3.py (hunk 1 above)
Comment (devinmatte) [reply server/chalicelib/s3.py:39]: I have this in the other branch https://github.com/transitmatters/t-performance-dash/pull/898
I guess I can have all three in this branch

**Review by idreyn — APPROVED** [review]
Lovely to see how well Gobble slots into our existing tools!

File: server/scripts/generate_line_files.py [hunk 3]
```diff
@@ -74,16 +92,42 @@ def cr_stop_info_to_s3_prefix(line, direction, stop_id):
...
+            stops_formatted.append(
+                {
+                    "stop_name": stop["attributes"]["name"],
+                    "station": stop["id"],
+                    "branches": None,
+                    "order": index + 1,
+                    "stops": stop_layout[stop["id"]],
+                }
+            )
+        except KeyError:
```
[the remark this answers is not in the document]
Comment (devinmatte) [reply server/scripts/generate_line_files.py:108]: So this was only occurring for Worcester, but I gave this a try for everything, and then many stops on some lines returned less possible stop ids than generating the list from s3. Meaning gobble is detecting the trains at stops that the MBTA api doesn't think they will stop at. So really what we need here is a merge imo

**Pull request merged**